module/testmodule/main.go

```diff
@@ -75,9 +75,9 @@ func newHelper(
 
 type helper struct {
 	resource.Named
-	resource.TriviallyReconfigurable
 	resource.TriviallyCloseable
-	logger logging.Logger
+	logger              logging.Logger
+	numReconfigurations int
```

Comment:
@maximpertsov `testmodule` now differs slightly from `testmodule2`, but I don't think that's an issue.